### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Ispirett**
Thanks for your time.

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -27,9 +27,9 @@ def initialize(p1, p2, d, game_manager)
     @current_turn = 9
   end
 
-
+  private
   def game_over?(_name = 'player')
```
Comment (inhaq) [inline lib/board.rb]: winning logic is hardcoded

**Review by inhaq — CHANGES_REQUESTED** [review]
Good work with folder structure and readme.
Update your game logic and try to separate your functions such that each function do one job. Also separate display messages from functionality.
I would recommend to have a look into `SOLID` and `OOP` concepts.
Once fixed, kindly resubmit for review.

File: lib/board.rb (hunk 1 above)
Comment (Ispirett) [reply lib/board.rb]: thanks

**Review by inhaq — CHANGES_REQUESTED** [review]
Nice work on updating and fixing some changes requested. Although winning logic still seems to be hardcoded, you can implement it by loops.

**Review by inhaq — CHANGES_REQUESTED** [review]
Great work, the code looks good. The only concern is when I play the game and then restart with same player names/values as the previous game then it acts weirdly(means either it show winner name directly or it just quit). Let me know of your thinking behind this if you guys are doing deliberately or it's a bug?

**Comment by SotoArmando** [comment]
This should be a bug, cause is expected to get a new board each time you play a new game.

**Comment by Ispirett** [comment]
Issue is fixed

**Review by inhaq — APPROVED** [review]
Great job, it's 👍🏻
In case I missed any issue, that'll be covered in tests. Go ahead and submit a project completion form.

**Comment by Ispirett** [comment]
Thanks !!!!

**Pull request merged**